### DISCUSSION TITLE
Help using ble-com-setup

### DISCUSSION THREAD

**Discussion by adamalfath** [Jakeler/ble-serial#86] [Q&A]
I have Windows 10, com0com & python 3.10 installed, executing the `ble-com-setup` command but just received `OK` response without creating link between  BLE and COM port. I caught a new terminal being opened but quickly closed again. `ble-scan` work and outputting some scanned device.

**Comment by Mboudali** [comment]
Can you see "com0com - serial port emulators" in device manager?
One of the ports should be "com0com - serial port emulator 2 (COM9 <-> BLE)".

**Comment by adamalfath** [comment]
So `ble-com-setup` command is still not working for me. I had to set it up manually on com0com utility then `ble-serial` worked as intended.
